index.html

```diff
@@ -280,11 +283,13 @@ <h2>Best Practices and Recommendations</h2>
         
         <p>Checking if a tag is <a>valid</a> requires access to or a copy of the registry plus additional runtime logic. While content authors are advised to choose, generate, and exchange only valid values, language tag matching and other common language tag operations are designed so that validity checking is not needed. Features or functions that need to understand the specific semantic content of subtags are the main reason that a specification would normatively require <a>valid</a> tags as part of the protocol or document format.</p>
         
+        <p class="advisement" id="ltli-canonical-req"><a class="self" href="#ltli-canonical-req">&#x200B;</a>Content authors and implementations SHOULD use language tags that are <a>canonical Unicode locale identifiers</a> and SHOULD normalize language tags that they consume using the rules for producing canonical tags.</p>
```

Comment:
The intention here was to avoid having a normative requirement placed into specifications. Mark commented that it would be best if the language tags found in the wild conformed to LDML's normalization requirements (which, it happens, are also what ECMA-402 is doing). So this is an advisement about that primarily. 

The contents of the -t and -u extensions I intend to be opaque to most content authors/implementations. But the additional canonicalization and tag choice rules in Sections 3.2 and 3.3 of TR35 are probably a good idea for content authors and implementations to hew to. The use of MAY is too weak. SHOULD is probably right, but, to your point, seems overly prescriptive. 

I should probably break this into two best practices: one for content authors and one for implementations. Let me try that next. Would appreciate your thoughts on how to address...

Reply:
"But the additional canonicalization and tag choice rules in Sections 3.2 and 3.3 of TR35 are probably a good idea for content authors and implementations to hew to. ": here I am not sure, see my comment below
https://github.com/w3c/ltli/pull/12#discussion_r454122073
ECMA-402 may be just more advanced than implementations like the W3C validator, but should we look into the details of the rules in section 3.2 and 3.3. to see what they mean for content authors and implementations? E.g. in section 3.2.1 it says
"Any script subtag is in title case (eg, Hant)"
is that included? 
About
"Would appreciate your thoughts on how to address"
How about listing the rules that we want to take into account from TR35? I know that normally you would just refer to the spec, but in this case I find it hard to identify the "meat" in TR35 relevant for implementers and content authors.

Reply:
TR35 is difficult to follow for sure. And it combines things that are in BCP47 (such as the case canonicalization rules) and it's own stuff. I think you're probably right. Let me survey further.